app/jobs/in_person/enrollments_ready_for_status_check/enrollment_pipeline.rb

```diff
@@ -73,11 +77,24 @@ def process_message(sqs_message)
         mail = Mail.read_from_string(ses_message[:content])
         # Depending on how the email is created, we may need to read different
         # parts of the message
-        if mail.multipart?
-          text_body = mail.text_part&.decoded
+        # Put all parts that are html/text encoded into a single value and extract
+        # the code from there, this will handle changes in the future to the email.
+
+        text_body = (mail.text_part || mail.html_part || mail).body.decoded
+
+        part_found = if mail.text_part
+                       'text_part'
+        elsif mail.html_part
+          'html_part'
         else
-          text_body = mail.body.decoded
+          'message_body'
         end
```

Comment:
Can we normalize the indentation here, either all left-aligned or all aligned with the equals (`=`) assignment? (Or extracting to a separate method)